esdoc/template/layout.html

```diff
@@ -20,9 +20,9 @@
   <div class="search-box active">
   <span>
     <!--Geops <img src="./image/search.png"/> -->
```

Comment:
I guess we can remove the commented code.

Reply:
I wanted to keep track of what was there before.